README.md

```diff
@@ -236,28 +207,19 @@ timeSeriesDf
 
 ### Data points
 
-Data points are always related to a time series. To read datapoints you will need to filter by a valid time series name,
-otherwise an empty DataFrame will be returned. For this reason it is important to be careful when using caching with
-this resource type.
+Data points are always related to a time series. To read data points you need to filter by a valid time series name, otherwise an empty DataFrame is returned. **Important**: be careful when using caching with this resource type.
```

Comment:
This is important because: if you read an existing time series A and cache it, and then try to filter for existing time series B, you'll get nothing. I think we should make the reason clearer, perhaps something like:
> be careful when using caching with this resource type. If you cache the result of a filter, and then apply another filter you will not trigger more data to be read from CDF, and thus end up with an empty DataFrame.